UPGRADING

```diff
@@ -742,7 +709,7 @@ PHP 8.4 UPGRADE NOTES
     Along with these, five constants (Pdo\Firebird::TRANSACTION_ISOLATION_LEVEL,
     Pdo\Firebird::READ_COMMITTED, Pdo\Firebird::REPEATABLE_READ,
     Pdo\Firebird::SERIALIZABLE, Pdo\Firebird::WRITABLE_TRANSACTION) have been added.
-  . When using persistent connections, there is now a liveness check in the
+  . When using persistent connections, there is now a liveliness check in the
```

Comment:
It seems to me that either is okay (https://www.merriam-webster.com/dictionary/liveness#:~:text=live%C2%B7%E2%80%8Bness,reverberant%20quality%20of%20a%20room).

Reply:
Well my IDE was complaining, I'll change it back.

Reply:
Not necessarily necessary. :) (My browser complains about almost every word I write; might enable English spellchecking at some point ;)